src/setup/rgw.py

```diff
@@ -537,7 +544,10 @@ def _update_hax_endpoint_and_create_admin(conf: MappedConf):
                     Rgw._get_cortx_conf(conf, f'node>{machine_id}>type') == 'data_node']
                 data_pod_hostnames.remove(current_data_node)
                 for data_pod_hostname in data_pod_hostnames:
-                    Rgw._update_hax_endpoint(conf, data_pod_hostname)
+                    try:
+                        Rgw._update_hax_endpoint(conf, data_pod_hostname)
+                    except Exception:
+                        break
```

Comment:
shouldnt 'continue' be there if exception occurs while getting hax endpoint

Reply:
continue will go ahead with creating rgw user.

Reply:
'continue' will go back to start of the 'for' loop, 'break' will infact deny further iterations with remaining data pods in the list

Reply:
My bad, little confused.. I had continue only in the beginning. 
Applied the changes